pyglider/ncprocess.py

```diff
@@ -67,6 +67,12 @@ def extract_timeseries_profiles(inname, outdir, deploymentyaml):
 
                     dss['v'] = dss.water_velocity_northward.mean()
                     dss['v'].attrs = profile_meta['v']
+                elif ('water_velocity_eastward' not in dss.keys()) and ('_FillValue' in profile_meta['u']):
```

Comment:
What is `'_FillValue'` usually?  I'm not familiar with how that gets set up...

Reply:
If this is _not_ meta data do we want to fill with NaN anyways, or just not have the variable at all?

Reply:
IOOS suggests -999 for a _FillValue, and it is specified in the variable-specific metadata for u and for v. 

I think if we don't specify it in the metadata, we probably don't want the variable at all. That allows people who don't care about e.g. submitting to IOOS to run the code without unused variables taking up space in their netcdf files.

Reply:
Thats fine.  Seems a little strange for FillValue to be a variable-specific metadata, but maybe different sensors need different fill values.